chia/consensus/constants.py

```diff
@@ -26,37 +28,37 @@ class ConsensusConstants:
     SUB_EPOCH_BLOCKS: uint32  # The number of blocks per sub-epoch
     EPOCH_BLOCKS: uint32  # The number of blocks per sub-epoch, must be a multiple of SUB_EPOCH_BLOCKS
 
-    SIGNIFICANT_BITS: int  # The number of bits to look at in difficulty and min iters. The rest are zeroed
-    DISCRIMINANT_SIZE_BITS: int  # Max is 1024 (based on ClassGroupElement int size)
-    NUMBER_ZERO_BITS_PLOT_FILTER: int  # H(plot id + challenge hash + signage point) must start with these many zeroes
-    MIN_PLOT_SIZE: int
-    MAX_PLOT_SIZE: int
-    SUB_SLOT_TIME_TARGET: int  # The target number of seconds per sub-slot
-    NUM_SP_INTERVALS_EXTRA: int  # The difference between signage point and infusion point (plus required_iters)
-    MAX_FUTURE_TIME: int  # The next block can have a timestamp of at most these many seconds more
-    NUMBER_OF_TIMESTAMPS: int  # Than the average of the last NUMBER_OF_TIMESTAMPS blocks
+    SIGNIFICANT_BITS: uint8  # The number of bits to look at in difficulty and min iters. The rest are zeroed
+    DISCRIMINANT_SIZE_BITS: uint16  # Max is 1024 (based on ClassGroupElement int size)
+    NUMBER_ZERO_BITS_PLOT_FILTER: uint16  # H(plot id + challenge hash + signage point) must start with these many zeroes # noqa: E501
+    MIN_PLOT_SIZE: uint16
+    MAX_PLOT_SIZE: uint16
+    SUB_SLOT_TIME_TARGET: uint32  # The target number of seconds per sub-slot
+    NUM_SP_INTERVALS_EXTRA: uint16  # The difference between signage point and infusion point (plus required_iters)
+    MAX_FUTURE_TIME: uint32  # The next block can have a timestamp of at most these many seconds more
```

Comment:
timestamps are uint64 in chia

Reply:
```
The next block can have a timestamp of at most these many seconds more
```

This is not a timestamp, its just an offset.